bin/gen-pgbouncer-conf.sh

```diff
@@ -19,9 +19,9 @@ cat >> "$CONFIG_DIR/pgbouncer.ini" << EOFEOF
 [pgbouncer]
 listen_addr = 127.0.0.1
 listen_port = 6000
-auth_type = md5
+auth_type = scram-sha-256
 auth_file = $CONFIG_DIR/users.txt
-server_tls_sslmode = prefer
```

Comment:
Do we expect this change to break anything? I suspect not now that we support TLS fleet-wide.

Reply:
We've seen a few GitHub Issues in the past where customers are using the buildpack to connect to RDS/Crunchy/etc. I hope that this change doesn't impact that as well but I would assume they are also supporting TLS and agree customers have bigger problems if they are not for some reason.